pipelines/ko-build-oci-ta/ko-build-oci-ta.yaml

```diff
@@ -170,7 +170,7 @@ spec:
       value:
       - $(tasks.build-container.results.IMAGE_URL)@$(tasks.build-container.results.IMAGE_DIGEST)
     - name: BUILDAH_FORMAT
-      value: docker
+      value: oci
```

Comment:
For consistency with the other tasks, do we want to comment that this is temporary? Some artrifacts building with this pipeline would still need to produce `docker` by default. 

This could just be a temporary change that you make locally until the index image is fixed.